common/src/main/java/org/keycloak/common/util/RandomString.java

```diff
@@ -0,0 +1,70 @@
+package org.keycloak.common.util;
```

Comment:
Don't we add licence headers anymore? ;)

Otherwise this looks okay.

Reply:
This is just a revert of the old file, so guess it didn't have a license file.